src/docs/metrics/open-interest/index.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Open Interest
+author: Ante
+date: 2023-11-28
+description: Total number of outstanding or open contracts in the market.
+---
+
+## Definition
+Open interest represents the total number of outstanding contracts that have not been settled by an offsetting trade. For example, if trader A buys a futures contract from trader B, and neither has closed out their position, the open interest is one.
+Traders can close out their positions by taking an opposing position (if they were long, they can sell; if they were short, they can buy). Changes in open interest can reflect whether traders are initiating new positions or closing existing ones.
```

Comment:
Minor suggestion for improvement:
- Original: `if they were long, they can sell; if they were short, they can buy`
- Suggested: `selling if they were long or buying if they were short`